packages/s/slim.js.json

```
@@ -15,13 +15,13 @@
     "binding"
   ],
   "autoupdate": {
-    "source": "git",
-    "target": "git://github.com/eavichay/slim.js.git",
+    "source": "npm",
+    "target": "slim.js",
     "fileMap": [
       {
         "basePath": "",
         "files": [
-          "Slim*"
+          "slim.js"
```

```suggestion
          "{,directives/}*.@(js|ts)"
```
The package contains quite a few files that the docs suggest are needed, even for use directly on a web page, so let's bring them all in. [Glob](https://www.digitalocean.com/community/tools/glob?comments=true&glob=%7B%2Cdirectives%2F%7D%2A.%40%28js%7Cts%29&matches=false&tests=%2F%2F%20Imported%20from%20slim-js%20NPM%20package&tests=package.json&tests=Decorators.d.ts&tests=Decorators.js&tests=jsconfig.json&tests=LICENSE&tests=README.md&tests=Slim.d.ts&tests=Slim.js&tests=Slim.nomodule.js&tests=.semaphore%2Fsemaphore.yml&tests=directives%2Fall.js&tests=directives%2Fall.nomodule.js&tests=directives%2Fbind-attr.js&tests=directives%2Fbind-attr.nomodule.js&tests=directives%2Fif.js&tests=directives%2Fif.nomodule.js&tests=directives%2Frepeat.js&tests=directives%2Frepeat.nomodule.js&tests=directives%2Fswitch-case.js&tests=directives%2Fswitch-case.nomodule.js&tests=src%2FDecorators.d.ts&tests=src%2FDecorators.js&tests=src%2Fdirectives%2Fall.js&tests=src%2Fdirectives%2Fbind-attr.js&tests=src%2Fdirectives%2Fif.js&tests=src%2Fdirectives%2Frepeat.js&tests=src%2Fdirectives%2Fswitch-case.js&tests=src%2FSlim.d.ts&tests=src%2FSlim.js)